### PR TITLE
Solutions Admin

### PR DESCRIPTION
**Pull request by Leusev**
### Purpose

Challenges's component Admin section for Solutions crud 

### :camera: Screenshots

### CODE REVIEW

File: app/models/decidim/solutions/solution.rb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      include Decidim::Publicable
+      include Decidim::Resourceable
+      include Decidim::ScopableComponent
+      include Decidim::Searchable
+      include Decidim::Traceable
+      include Decidim::TranslatableAttributes
+
+      # FIX define possible requirements !
+      VALID_REQUIREMENTS = [:technical, :economic, :financial].freeze
+      enum requirements: VALID_REQUIREMENTS
```
Comment (tramuntanal) [inline app/models/decidim/solutions/solution.rb:18]: This is not an enum but a free text field

File: db/migrate/20201116111200_create_decidim_solutions_solutions.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    create_table :decidim_solutions_solutions do |t|
+      t.jsonb :title
+      t.jsonb :description
+      t.references :decidim_component, index: true, null: false
+      t.references :decidim_problems_problem, index: { name: "decidim_challenges_problems_solutions" }, null: false
+      t.references :decidim_scope, index: true
+      t.jsonb :tags
+      t.string :indicators
+      t.string :beneficiaries
+      t.integer :requirements, null: false, default: 0
```
Comment (tramuntanal) [inline db/migrate/20201116111200_create_decidim_solutions_solutions.rb:14]: @Leusev This is not an enum but a multilanguage free text field
```suggestion
      t.jsonb :requirements
```

File: db/migrate/20201116111200_create_decidim_solutions_solutions.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+      t.string :financing_type
```
Comment (tramuntanal) [inline db/migrate/20201116111200_create_decidim_solutions_solutions.rb:15]: All text fields should be multi-language

**Review by tramuntanal — CHANGES_REQUESTED** [review]

File: app/models/decidim/solutions/solution.rb (hunk 1 above)
Comment (Leusev) [reply app/models/decidim/solutions/solution.rb:18]: oh, ok @tramuntanal ! I'll change it then!

File: db/migrate/20201116111200_create_decidim_solutions_solutions.rb (hunk 3 above)
Comment (Leusev) [reply db/migrate/20201116111200_create_decidim_solutions_solutions.rb:15]: ok, I'll fix it

**Review by tramuntanal — CHANGES_REQUESTED** [review]
@Leusev I've left you some comments, can you check please?

**Review by tramuntanal — APPROVED** [review]

**Pull request merged**